awx/api/views/metrics.py

```diff
@@ -37,6 +38,10 @@ class MetricsView(APIView):
     def get(self, request):
         ''' Show Metrics Details '''
         if (request.user.is_superuser or request.user.is_system_auditor):
-            return Response(metrics().decode('UTF-8'))
+            metrics_to_show = ''
+            if 'redisonly' not in request.query_params:
```

Comment:
nitpick, but wonder if we don't want o directly mention redis here in the query because 1) its an implementation detail maybe users don't know and 2) maybe we'll change out redis someday 

Is there some other way to describe these metrics like....`callback_receiver` since that is what ll the metrics are referencing?

so it would be like `/api/v2/metrics?callback_receiver`